### ISSUE TITLE
Boardle doesn't respond

### ISSUE THREAD

**Issue by Zonkil9** [alexberryman/DiscordWordle#28]
Hello.
Boardle doesn't respond today on any command although it is online. Can you fix it?

**Comment by alexberryman** [comment]
Thanks for reporting the issue. The bot was able to see and track all of the scores, but was unable to respond. Looks like some kind of rate limit or transient error:

```
Mar 08 07:29:53 [discordle] 2022/03/08 13:29:52 [DG0] 
restapi.go:154:RequestWithLockedBucket() rate limit unmarshal
 error, invalid character 'e' looking for beginning of value
```

Restarting the bot seemed to resolve the issue. I'll dig deeper into this error and try to prevent it in the future.

**Comment by Zonkil9** [comment]
Today it doesn't respond again. Another restart needed?

**Comment by alexberryman** [comment]
What commands are you trying to run? I'm seeing a few "out of memory" errors before you reported this:

```
Apr 10 03:45:05 [discordle] [heroku/bot.1] Process running mem=585M(114.4%)
Apr 10 03:45:05 [discordle] [heroku/bot.1] Error R14 (Memory quota exceeded)
Apr 10 03:45:28 [discordle] [heroku/bot.1] Process running mem=591M(115.5%)
Apr 10 03:45:28 [discordle] [heroku/bot.1] Error R14 (Memory quota exceeded)
```

Are you trying to view the scoreboard of a massive server, or upload a quip image or anything out of the ordinary?

**Comment by Zonkil9** [comment]
Nope, there were just ordinary score inputs. There are only seven people on this server. I'll try to explain exactly what happened:

> 10:02 CET: Person 1 inputs their score. No response from Boardle.
> 10:09 CET: Person 2 inputs their score. No response either.
> 10:11 CET: I write here on GitHub that Boardle doesn't respond.
> 10:18 CET: Person 3 inputs their score. No response either.
> 10:45 CET: Suddenly Boardle wakes up. It answers with three quips at once and records our scores.
> 10:46 CET: I call @Boardle scoreboard and it responds normally showing our scores.
> 11:59 CET: Person 4 inputs their score. Again, no response from Boardle.
> 13:28 CET: I call @Boardle scoreboard to see if it woke up and yes, it works.
> 13:39 CET: Person 4 inputs their score again. This time Boardle works fine and it records the score.

Hope that helps.

**Comment by alexberryman** [comment]
That is helpful. I'll step through the logs and see if there is anything I can do on my side to prevent this from happening again.

The fact that it sat on the replies for ~30min is interesting.
